scripts/pages/multi-ship-checkout.js

```diff
@@ -169,6 +169,129 @@ require(["modules/jquery-mozu",
         }
     });
 
+    var PickupView = Backbone.MozuView.extend({
+        templateName: 'modules/multi-ship-checkout/pickup-contact',
+        renderOnChange: [
+            'isReady'
+        ],
+        initialize: function() {
+
+            var alternatePickupContact= new AlternatePickupView({
+                el:$('#alternate-contact'),
+                model: this.model.get("alternateContact")
+            });
+           
+        },
+        render: function() {
+            var self = this;
+            Backbone.MozuView.prototype.render.apply(this,arguments);
+                setTimeout(
+                    function() {
+                        var alternatePickupContact= new AlternatePickupView({
+                            el:$('#alternate-contact'),
+                            model: self.model.get("alternateContact")
+                        });
```

Comment:
Why deuplicate initialization when already done in initialize()?

Reply:
binding of the click events of child view are getting removed when we call render in parent function. so recreating child view in render of parent view.